quay-enterprise/app-tokens.md

```diff
@@ -1,15 +1,15 @@
 # Quay Enterprise Application Tokens #
 
-Since Quay Enterprise version 2.7.0, a new way of authenticating to Quay from Docker CLI (and other apps) was added, in the form of application tokens. These tokens don't expire by default, but this expiration can be set inside the Quay superadmin control panel. 
-To enable expiration of application tokens, the following option must be selected in the Quay Enterprise suerpadmin control panel:
+Since Quay Enterprise version 2.7.0, a new way of authenticating to Quay from Docker CLI (and other apps) was added, in the form of application tokens. These tokens don't expire by default, but this expiration can be set inside the Quay super user control panel. 
+To enable expiration of application tokens, the following option must be selected:
 
 ![Quay Enterprise superadmin control panel](https://github.com/ibazulic/docs/blob/master/quay-enterprise/img/set-token-expiration-time.png "Quay Enteprrise admin control panel")
 
 Note that the number of days **needs** to be specified, if the text field is blank, the tokens will never expire.
 
 ## Icons explained ##
 
-In a typical situation, a repo owner might have multiple tokens created for multiple applciations, and every token has its own expiry time, as we can see here:
+In most cases, a repo owner will have multiple tokens for multiple applications and each token will have its own epiry time. 
```

Comment:
Don't use `repo owner`. These are `namespace owners`. Also, `expiry` is misspelled, and `expiration` is preferred